packages/generators/add-generator.ts

```diff
@@ -140,7 +140,7 @@ export default class AddGenerator extends Generator {
 							.then(
 								(entryTypeAnswer: { entryType: boolean }): Promise<void | {}> => {
 									// Ask different questions for entry points
-									return entryQuestions(self, entryTypeAnswer);
+									return entryQuestions(self, entryTypeAnswer.entryType);
```

Comment:
I think that the entrytype here is already module destructed, no?

Reply:
Not really, `entryTypeAnswer: { entryType: boolean }`  is a type annotation :)